src/styles/stylesheet.css

```diff
@@ -6,6 +6,24 @@ body {
   --primary-color: orange;
 }
 
+.footer-wrapper {
+  width: 60%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 22px;
+  margin: 0 auto;
+}
+
+.footer__icon {
+  border-style: solid;
+  border-color: #404040;
+  border-radius: 50%;
+  padding: 5px 10px;
+  margin: 0;
+  display: flex;
+  justify-content: space-evenly;
+}
```

Comment:
Consider set static `width` and `height` to make sure we have square, not an oval. Also set after this `aling-items: center` to align icons in squares.